tests/Doctrine/Tests/DBAL/Functional/PDOStatementTest.php

```diff
@@ -28,10 +28,6 @@ protected function setUp() : void
         $this->connection->getSchemaManager()->dropAndCreateTable($table);
     }
 
-    /**
-     * @group legacy
```

Comment:
`@group` is from PHPUnit, though.

Reply:
Yeah, but `legacy` isn't.

Reply:
I thought we were using that group to identify the tests we were going to drop eventually because of legacy :laughing:

Reply:
Wouldn't be a bad idea, but right now I bet it's far from being exhaustive :P